radio/src/gui/colorlcd/model_inputs.cpp

```diff
@@ -315,9 +315,15 @@ void CommonInputOrMixButton::drawFlightModes(BitmapBuffer *dc,
                                              FlightModesType value,
                                              LcdFlags textColor)
 {
+  coord_t startX;
+  if (LCD_W > LCD_H)
+    startX = 146;
+  else
+    startX = 136;
+
   dc->drawMask(146, 2 + PAGE_LINE_HEIGHT + FIELD_PADDING_TOP,
                mixerSetupFlightmodeIcon, textColor);
-  coord_t x = 166;
+  coord_t x = startX + 20;//166;
```

Comment:
What is the reason for this not being the startX value? (in other words, why isn't `startX` 166/156?)

Reply:
Ah, ok, your line 324 should be startX also, not 146 ;) You can get rid of the `//166` also as that is in the git history if it's ever wanted to be seen again.

Reply:
Thank you! You are right. I was just trying to minimize the changes. :-)